src/test/run-fail/test-panic.rs

```diff
@@ -12,6 +12,7 @@
 // error-pattern:thread 'test_foo' panicked at
 // compile-flags: --test
 // ignore-pretty: does not work well with `--test`
+// ignore-emscripten
```

Comment:
This one still doesn't seem to work. Message isn't printed. Seems likely emscripten's stdio isn't wired up to node's.